File: geometry/volume_formulas/objc/volume_formulas.m
- (CGFloat)cubeVolumeWithSide:(CGFloat)side {
    return pow(side, 3.0);
}

- (CGFloat)rightRectangularPrismVolumeWithLength:(CGFloat)length width:(CGFloat)width height:(CGFloat)height {
    return length * width * height;
}

- (CGFloat)coneVolumeWithBaseRadius:(CGFloat)radius height:(CGFloat)height {
    CGFloat baseArea = M_PI * pow(radius, 2.0);
    return height * baseArea / 3.0;
}

- (CGFloat)cylinderVolumeWithBaseRadius:(CGFloat)radius height:(CGFloat)height {
    CGFloat baseArea = M_PI * pow(radius, 2.0);
    return height * baseArea;
}

- (CGFloat)sphereVolumeWithRadius:(CGFloat)radius {
    return (4.0 / 3.0) * M_PI * pow(radius, 3.0);
}

File: geometry/volume_formulas/java/VolumeFormulas.java
public class VolumeFormulas {

    public double getConeVolume(float radius, double height) {
        return (Math.PI*height*(radius*2))/3;
    }

    public double getCubeVolume(double edgeLenght) {
        return edgeLenght*3;
    }

    public double getCuboidVolume(double width, double lenght, double height) {
        return width*lenght*height;
    }

    public double getCylinderVolume(float radius, double height) {
        return Math.PI*height*(radius*2);
    }

    public double getPyramidVolume(double width, double lenght, double height) {
        return (width*lenght*height)/3;
    }

    public double getSphereVolume(float radius) {
        return ( 4.0 / 3.0 ) * Math.PI * Math.pow( sphereRadius, 3 );
    }


}
